guides/common/modules/con_best-practices-for-role-based-access-control-in-project.adoc

```diff
@@ -0,0 +1,22 @@
+[id="best-practices-for-role-based-access-control-in-{project-context}_{context}"]
+= Best practices for role-based access control in {Project}
+
+* Define the expected tasks and responsibilities:
+Define the subset of the {Project} infrastructure that you want the role to access as well as actions permitted on this subset.
+Think of the responsibilities of the role and how it differs from other roles.
+
+* Use predefined roles whenever possible:
+{Project} provides a number of sample roles that can be used alone or as part of a role combination.
+Copying and editing an existing role can be a good start for creating a custom role.
+
+* Add permissions gradually and test the results:
+When creating a custom role, start with a limited set of permissions and add permissions one by one, while testing continuously.
+Make sure to test your custom role to verify that it works as intended.
+
+* Consider all affected entities:
+For example, a content view promotion automatically creates new Puppet Environments for the particular lifecycle environment and content view combination.
```

Comment:
uhm. I do not understand this. Can someone please confirm/ ACK?

Reply:
Let's just drop it. The point above (about testing role-based access thoroughly) covers this too.